sensors/aidl/Sensor.cpp

```diff
@@ -233,10 +274,32 @@ std::vector<Event> Sensor::readEvents() {
                     .z = 50.0,
                     .status = SensorStatus::ACCURACY_HIGH,};
                 event.payload.set<EventPayload::Tag::vec3>(vec3);
+                events.push_back(event);
+                for (const auto& frame : mAdditionalInfoFrames) {
```

Comment:
We use the same code base for CIV as well, and additional info may not be needed for CIV.
Can we add this code in condition based on TARGET_PRODUCT?

Reply:
yes this can be done to support target specific features.